frontend/openchat-client/src/openchat.ts

```diff
@@ -3501,16 +3501,23 @@ export class OpenChat extends EventTarget {
             };
             const avatarId = this._liveState.userStore[this.user.userId]?.blobReference?.blobId;
             const chatsResponse =
-                this._chatUpdatesSince === undefined
-                    ? await this.initialState()
-                    : await this.updates(this._chatUpdatesSince, currentState, avatarId);
-
-            this._chatUpdatesSince = chatsResponse.timestamp;
+                this._latestUserCanisterUpdates === undefined
+                    ? await this.api.getInitialState()
```

Comment:
So you mean we always call `getUpdates` with no args, we then get latest from the cache and call either `updates` or `initial_state` depending on if there is something in the cache or not?

Reply:
Yeah - we kind of already have this kind of logic in `openchatAgent` already and it would see nice to keep it all in there and then just not have any sort of decision to make inside the OpenChat client itself.

Reply:
Then maybe we can completely remove either `getUpdates` or `getInitialState` from the client and the worker interface.